src/styles/components/_about-us-section.scss

```diff
@@ -1,6 +1,12 @@
+@import '../abstracts/_abstracts-dir';
+
 .about-us-section {
+  padding: rem(24px) 0;
   box-sizing: border-box;
-  padding: rem(16px) 0;
+
+  @include respond-above('sm') {
+    padding: rem(32px) 0;
+  }
```

Comment:
Screen size within `sm` - `md` shouldn't have bigger `paddings` than `< sm`. Please use the same values from `home-section`.